test/specs/collections/Table/TableCell-test.js

```diff
@@ -42,5 +42,11 @@ describe('TableCell', () => {
 
       wrapper.text().should.equal(content)
     })
+
+    it('render 0 value', () => {
```

Comment:
I moved this to commonTests.rendersChildren(). I am not sure if this check:
`if (Component._meta.name !== 'Embed') {...}`

is ok, but I need it because `<Embed>{0}</Embed>` is rendered as `<Icon/>`